packages/discord.js/typings/index.d.ts

```diff
@@ -6094,7 +6136,7 @@ export interface InteractionReplyOptions extends BaseMessageOptionsWithPoll {
   >;
 }
 
-export interface InteractionUpdateOptions extends MessageEditOptions {
+export interface InteractionUpdateOptions extends MessageEditOptions, SharedInteractionResponseOptions {
   fetchReply?: boolean;
```

Comment:
Let's mark fetchReply as deprecated while we're at it

Reply:
Why deprecate this and not remove it?

Reply:
We can remove it since this targets main and make it sem major, with backport deprecating it